iocAdmin/Db/Makefile

```diff
@@ -36,3 +41,7 @@ include $(TOP)/configure/RULES
 #----------------------------------------
 #  ADD RULES AFTER THIS LINE
 
+$(COMMON_DIR)/iocAdminRTEMS.db:	$(INSTALL_DB)/iocAdminScanMon.db $(INSTALL_DB)/epicsEnvVars.db
+
+$(COMMON_DIR)/iocAdminSoft.db:	$(INSTALL_DB)/iocAdminScanMon.db $(INSTALL_DB)/epicsEnvVars.db
+
```

Comment:
Is there some reason that `iocAdminVxWorks` is not included here?

Reply:
I just overlooked it as we no longer deploy vxWorks here at SLAC.